beacon-chain/node/node.go

```diff
@@ -843,7 +836,7 @@ func (b *BeaconNode) registerRPCService() error {
 	return b.services.RegisterService(rpcService)
 }
 
-func (b *BeaconNode) registerPrometheusService(cliCtx *cli.Context) error {
+func (b *BeaconNode) registerPrometheusService(_ *cli.Context) error {
```

Comment:
why this change?

Reply:
cliCtx isn't used in this so usually deep source complains when you touch the file and there's unused arguments